crates/core_arch/src/simd.rs

```diff
@@ -186,9 +187,20 @@ simd_ty!(
 simd_ty!(i32x4[i32]: x0, x1, x2, x3);
 simd_ty!(i64x2[i64]: x0, x1);
 
+simd_ty!(
+    #[unstable(feature = "f16", issue = "116909")]
```

Comment:
We don't need stability attributes for these since they are private types and are only used internally.